src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -73,6 +73,12 @@ class ServiceTypeAhead extends Component {
         onChange={this.handleOnSelect}
         defaultSelectedItem={defaultSelectedItem}
         itemToString={renderService}
+        onInputValueChange={(inputValue, stateAndHelpers) => {
+          const { selectedItem, clearSelection } = stateAndHelpers;
+          if (selectedItem && inputValue !== selectedItem.name.trim()) {
```

Comment:
No idea why but the selectedItem name can have a ton of whitespace at the end of it.